manifest_staging/charts/secrets-store-csi-driver/templates/role-syncsecrets_binding.yaml

```diff
@@ -0,0 +1,14 @@
+{{ if .Values.syncSecrets.install }}
```

Comment:
We also need to update the `manifests` target in `Makefile` so we can automate this - https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/master/Makefile#L160-L170

Reply:
Ah okay I added the files to `config/rbac`, edited and ran the makefile. So now the stuff in `manifest_staging` is all the _output_ of the `make manifests`

Reply:
Ah `make manifests` is modifying `config/rbac/role.yaml` and reverting my changes there. I'll need to figure that out.